client/client.go

```diff
@@ -324,24 +324,13 @@ func Configure(logger hclog.Logger, providerConfig interface{}) (schema.ClientMe
 		// This is a try to solve https://aws.amazon.com/premiumsupport/knowledge-center/iam-validate-access-credentials/
 		// with this https://github.com/aws/aws-sdk-go-v2/issues/515#issuecomment-607387352
 		switch {
```

Comment:
maybe remove this switch and just use standard `if` and `else`

Reply:
I was using a switch whenever there was more than 3 or more cases and a if/else when there were fewer than 3 cases.